packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/dataStreamSearch/workspaceSelector/workspaceSelector.tsx

```diff
@@ -41,7 +57,10 @@ export const WorkspaceSelector = ({ control, client, OnGettingError }: Workspace
             disabled={status == 'error'}
             options={workspaceOptions}
             selectedOption={field.value}
-            onChange={({ detail }) => field.onChange(detail.selectedOption)}
+            onChange={({ detail }) => {
```

Comment:
probably having a onChangeHandler function instead of inline

Reply:
Added onChangeHandler